src/main/java/com/elixr/poc/constants/ApplicationConstants.java

```diff
@@ -3,20 +3,20 @@
 /**
  * This constant file holds all the error messages for all the end point
  */
+
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class ApplicationConstants {
 
+    public static final String AMOUNT_MISSING_IN_PURCHASE = "Amount is mandatory";
```

Comment:
Its better to add validation messages in the message.properties file.

Reply:
Updated